wiki/Ranking_Criteria/osu!/en.md

```diff
@@ -78,6 +78,7 @@ Overall rules and guidelines apply to every kind of osu! difficulty. Rhythm rela
 -   **Avoid using similar combo colours in successive combos.** Players should be able to differentiate between different combos in a beatmap.
 -   **Spinner ends, slider ends, and slider reverses should have hit sound feedback.** If these are used to represent a held sound and do not align with a distinct sound, having no feedback is acceptable.
 -   **Avoid using sound samples for sliderslide, sliderwhistle, and spinnerspin which do not naturally loop.** These hit sounds are continuous, meaning that their files play from start to end and loop as one continuous sound for the length of the object, so using sound files with a clear impact for them might lead to unwanted side effects.
+-   **Avoid replacing the hit finish in soft/normal samplesets with frequently used custom hit sound samples.** This can sound obnoxious for anyone disabling beatmap hit sounds. Replacing hit whistles/claps is recommended because those samples are used more often.
```

Comment:
nitpicky but this can be literally interpreted to not use hitsounds that are popular and that using less popular hitsounds are ok, unsure if change is needed

Reply:
second sentence's changes should clarify that a bit better now